flex/views/experiment/TabSamplesIllumina.mxml

```diff
@@ -206,816 +207,824 @@
+				Alert.show("Maximum allowed limit is 30 characters");
 			}
-					
-			private function defaultToZero(item:Object, column:AdvancedDataGridColumn):String{
-				if(item[column.dataField] == ''){
-					item[column.dataField] = '0';
-					return item[column.dataField];
-				} else{
-					return item[column.dataField];
-				}
-				
+		}
+
+		private function numberSequencingLanesColTip(item:Object):String {
+			return "This is the number of times (1 or greater) that you want to sequence this sample.";
+		}
+
+		private function defaultToZero(item:Object, column:AdvancedDataGridColumn):String {
+			if (item[column.dataField] == '') {
+				item[column.dataField] = '0';
+				return item[column.dataField];
+			} else {
+				return item[column.dataField];
 			}
-			
-			private function instructionLink(event:TextEvent):void {
-				if (event.text == 'downloadSampleSheet') {
-					downloadSampleSheet();
-				} else if (event.text == 'uploadInstructions') {
-					PopUpManager.centerPopUp(PopUpManager.createPopUp(this, UploadSampleSheetInstructionsView, true));
-				}
+
+		}
+
+		private function instructionLink(event:TextEvent):void {
+			if (event.text == 'downloadSampleSheet') {
+				downloadSampleSheet();
+			} else if (event.text == 'uploadInstructions') {
+				PopUpManager.centerPopUp(PopUpManager.createPopUp(this, UploadSampleSheetInstructionsView, true));
 			}
-			
-			public function changeConcentrationUnit(selectedUnit:String):void{
-				for each(var s:XML in samplesGrid.dataProvider) {
-					s.@codeConcentrationUnit = selectedUnit;
-					
-				}
+		}
+
+		public function changeConcentrationUnit(selectedUnit:String):void {
+			for each(var s:XML in samplesGrid.dataProvider) {
+				s.@codeConcentrationUnit = selectedUnit;
+
 			}
-			
+		}
+
 		]]>
 	</mx:Script>	
 	<mx:Sort id="barcodeSort"  compareFunction="sortBarcodes"/>
 	<mx:XMLListCollection id="barcodes" source="{parentApplication.dictionaryManager.xml.Dictionary.(@className=='hci.gnomex.model.OligoBarcode').DictionaryEntry}" sort="{barcodeSort}"/>
 	<mx:XMLList id="barcodeList"/>
-	
+
 	<mx:VBox width="100%" height="100%" verticalGap="1" paddingBottom="2" paddingLeft="4" paddingRight="4" paddingTop="2">
-	
-		
+
+		<mx:VBox verticalGap="5">
+			<mx:TextArea styleName="instructionBox" width="1230" wordWrap="true" id="textarea1" height="120" editable="false" verticalScrollPolicy="off" fontSize="11">
+				<mx:htmlText><![CDATA[1.  Assign a multiplex group number for each sample in the table below. Samples that are to be sequenced in the same lane should be assigned the same multiplex group number.
+2.  Provide a name for each sample.
+3.  Provide the concentration for each sample.
+4.  Type the volume (ul) that will be provided for each sample.
+5.  Specify the number of sequence lanes required for each multiplex group.  (once multiplex groups are selected, make gnomex so that the user can only enter one number per multiplex group and not a number for each sample).

```

Comment:
The stuff in between parentheses are instructions from Brian.  Let's go ahead and remove those from this box.

I will make this a JIRA issue